projects/pumpspace/index.js

```diff
@@ -0,0 +1,146 @@
+// projects/pumpspace/index.js
+const ADDRESSES = require('../helper/coreAssets.json')
+const { staking } = require('../helper/staking')
+const { getUniTVL } = require('../helper/unknownTokens');
+const { getTridentTVL } = require('../helper/sushi-trident')
+const { sumTokens2 } = require('../helper/unwrapLPs')
+const sdk = require('@defillama/sdk')
+
+const CHAIN = 'avax'
+
+// --- FACTORIES / CONTRACTS ---
+const PUMP_FACTORY   = '0x26B42c208D8a9d8737A2E5c9C57F4481484d4616' // V2
+const PUMP_V3        = '0xE749c1cA2EA4f930d1283ad780AdE28625037CeD' // V3/Trident
+
+// If you later expose staking for other MasterChefs, add here
+const MASTERCHEFS = [
+  '0x40a58fc672F7878F068bD8ED234a47458Ec33879', // SHELL
+  '0x56b54a1384d35C63cD95b39eDe9339fEf7df3E42', // KRILL
+  '0x06C551B19239fE6a425b3c45Eb8b49d28e8283C6', // PEARL
+]
+
+// --- TOKENS (project/local wrappers + protocol tokens) ---
+const TOKENS = {
+  bUSDT: '0x3C594084dC7AB1864AC69DFd01AB77E8f65B83B7',  // mapped to USDt
+  WAVAX_PROXY: "0xAB4fBa02a2905a03adA8BD3d493FB289Dcf84024",  // mapped to canonical WAVAX below
+  sBWPM: "0x6c960648d5F16f9e12895C28655cc6Dd73B660f7",
+  sADOL: "0x6214D13725d458890a8EF39ECB2578BdfCd82170",
+  CLAM:  "0x1ea53822f9B2a860A7d20C6D2560Fd07db7CFF85",
+  PEARL: "0x08c4b51e6Ca9Eb89C255F0a5ab8aFD721420e447",
+  KRILL: "0x4ED0A710a825B9FcD59384335836b18C75A34270",
+  SHELL: '0xaD4CB79293322c07973ee83Aed5DF66A53214dc6',
+}
+
+
+// Map wrapped/local tokens to canonical core assets for pricing/aggregation.
+// - bUSDT → USDt (Tether on Avalanche)
+// - WAVAX proxy (0xAB4f...) → canonical WAVAX (0xB31f...) 
+function transformAddress(addr) {
```

Comment:
we will add support for these assets so this transform will no longer be required https://github.com/DefiLlama/defillama-server/pull/10877